### ISSUE TITLE
BUG: hull() fails with TypeError

### ISSUE THREAD

**Issue by z3dev** [Spiritdude/OpenJSCAD.org#114]
function main() {
// BUG: If resolution is changed to 18 then there's an error
// "TypeError: this.points[p1] is undefined"
    var r = 18;
    var a = CAG.circle({center: [-5,0], radius: 3, resolution: r});
    var b = CAG.circle({center: [+5,0], radius: 3, resolution: r});
    b = hull(a,b);
    return b;
}

**Comment by z3dev** [comment]
This issue was moved to jscad/scad-api#4

**Comment by ahdinosaur** [comment]
hi, i believe this bug still exists. :bug: 

example:

```js
const modeling = require('@jscad/modeling')

const { circle } = modeling.primitives
const { hull } = modeling.hulls
 
const main = () => {
  var r = 18
  var a = circle({ center: [-5,0], radius: 3, segments: r })
  var b = circle({ center: [+5,0], radius: 3, segments: r })
  return hull(a, b)
}
 
module.exports = { main }
```

**Comment by hrgdavor** [comment]
fixed in #897  @ahdinosaur check it out if it works for you

![image](https://user-images.githubusercontent.com/2480762/128060565-a43304b8-26d6-4f8d-bcd0-5e7700cc5dfd.png)

**Comment by z3dev** [comment]
@ahdinosaur the latest versions of openjscad.xyz (and libraries) have the fix for this issue. please try out the latest, and let us know if there are further issues. thanks
